ext/http/response_body.rs

```diff
@@ -546,71 +514,46 @@ impl PollFrame for BrotliResponseStream {
 
     let res = match frame {
       ResponseStreamResult::NonEmptyBuf(buf) => {
-        let mut output_written = 0;
-        let mut total_output_written = 0;
-        let mut input_size = buf.len();
-        let input_buffer = buf.as_ref();
-        let mut len = max_compressed_size(input_size);
-        let mut output_buffer = vec![0u8; len];
-        let mut ob_ptr = output_buffer.as_mut_ptr();
-
-        // SAFETY: these are okay arguments to these FFI calls.
-        unsafe {
-          brotli::ffi::compressor::BrotliEncoderCompressStream(
-            this.stm.stm,
-            brotli::ffi::compressor::BrotliEncoderOperation::BROTLI_OPERATION_PROCESS,
```

Comment:
Removed the `BROTLI_OPERATION_PROCESS` call here. Tests still pass locally but I'm not sure if this is correct.

Reply:
That seems redundant